apps/jetpack-cloud/webpack.config.js

```diff
@@ -0,0 +1,149 @@
+/**
+ **** WARNING: No ES6 modules here. Not transpiled! ****
+ */
+/* eslint-disable import/no-nodejs-modules */
+
+/**
+ * External dependencies
+ */
+const _ = require( 'lodash' );
+const FileConfig = require( '@automattic/calypso-build/webpack/file-loader' );
+const getBaseWebpackConfig = require( '@automattic/calypso-build/webpack.config.js' );
+const path = require( 'path' );
+const SassConfig = require( '@automattic/calypso-build/webpack/sass' );
+const TranspileConfig = require( '@automattic/calypso-build/webpack/transpile' );
+
+/**
+ * Internal variables
+ */
+const cacheIdentifier = require( '../../server/bundler/babel/babel-loader-cache-identifier' );
+const isDevelopment = process.env.NODE_ENV !== 'production';
+
+const nodeModulesToTranspile = [
```

Comment:
This is borrowed from Calypso's webpack config file. Ideally, this list should be narrowed down or removed completely, or if we need it, we should define it in a single place and reuse it in both places.